saleor/dashboard/order/forms.py

```diff
@@ -307,3 +310,29 @@ class OrderFilterForm(forms.Form):
 
 class PaymentFilterForm(forms.Form):
     status = forms.ChoiceField(choices=PAYMENT_STATUS_CHOICES)
+
+
+class CreateOrderForm(forms.ModelForm):
+    user = AjaxSelect2ChoiceField(queryset=User.objects.all())
+
+    class Meta:
+        model = Order
+        fields = ('user',)
+
+    def __init__(self, *args, **kwargs):
+        super(CreateOrderForm, self).__init__(*args, **kwargs)
+        self.fields['user'].set_fetch_data_url(
+            reverse('dashboard:ajax-users-list'))
+
+    def save(self, commit=True):
+        user = self.instance.user
+        self.instance.user_email = user.email
+        self.instance.shipping_address = user.default_shipping_address
+        billing_address = user.default_billing_address
+        if not billing_address:
+            billing_address = Address.objects.create(
+                country=settings.DEFAULT_COUNTRY)
```

Comment:
Should we pass user's first name and last name here as well?

Reply:
That does not look right. How useful is an empty address object?

Reply:
You are both right. I think we could allow from now to store null as billing_address (for purpose of creating orders from dashboard).

Reply:
Right, I see two options here: either ask the admin to provide a valid address right away or allow orders to have missing addresses. The second is easier to implement but we should make sure that all actions like payment and shipping are blocked until a valid address is provided.